benchmarks/graphics_pipeline/GraphicsBenchmarkApp.h

```diff
@@ -436,11 +437,11 @@ class GraphicsBenchmarkApp
     float3                            mLightPosition       = float3(10, 250, 10);
     std::array<bool, TOTAL_KEY_COUNT> mPressedKeys         = {0};
     bool                              mEnableMouseMovement = true;
-    uint64_t                          mGpuWorkDuration;
+    RealtimeValue<uint64_t, float>    mGpuWorkDuration;
```

Comment:
the 2nd `float` in this declaration is not needed?

Reply:
I'd rather keep float there, since `uint64_t` -> `float` will lose quite a few digits of precision (double is better but it's still not exact value).